manifests/init.pp

```diff
@@ -644,8 +644,20 @@
   $gr_ldap_options                        = {},
   $gr_use_remote_user_auth                = 'False',
   $gr_remote_user_header_name             = undef,
-  $gr_rrd_dir                             = '/opt/graphite/storage/rrd',
-  $gr_local_data_dir                      = '/opt/graphite/storage/whisper',
+  $gr_base_dir                            = '/opt/graphite',
+  $gr_storage_dir                         = "${gr_base_dir}/storage",
+  $gr_local_data_dir                      = "${gr_storage_dir}/whisper",
+  $gr_rrd_data_dir                        = "${gr_storage_dir}/rrd",
```

Comment:
We should name it `$gr_rrd_dir`. This merge will be transparent for old users and will not break configurations.